eng/testing/AppleRunnerTemplate.sh

```diff
@@ -1,11 +1,9 @@
 ﻿#!/usr/bin/env bash
 
 EXECUTION_DIR=$(dirname $0)
-TEST_NAME=$1
-TARGET_ARCH=$2
-TARGET=
-SCHEME_SDK=
+[[RunCommands]]
```

Comment:
I'm trying to get rid of the runcommands templating mechanism as having individual runner scripts for different targets (ios/android/wasm) seems to be sufficient. Doing that in https://github.com/dotnet/runtime/pull/39923. That will basically undo this change here. Any objections?

Reply:
No, I support that change as we should be able to define it via MSBuild.

Reply:
No objections from me.